src/mlpack/tests/callback_test.cpp

```diff
@@ -190,6 +192,23 @@ BOOST_AUTO_TEST_CASE(NCAWithOptimizerCallback)
   BOOST_REQUIRE_GT(stream.str().length(), 0);
 }
 
+/**
+ * Test softmax_regression implementation with PrintLoss callback.
+ */
+BOOST_AUTO_TEST_CASE(SRWithOptimizerCallback)
```

Comment:
This Test is failing. I think `softmaxRegression.Train()` is trying to access Invalid location of `data`